docs/schema/codelists.md

```diff
@@ -40,16 +40,14 @@ The organizations, economic operators or other participants in a contracting pro
 :file: ../../build/current_lang/codelists/partyRole.csv
 ```
 
-### Item Classification Scheme
+### Classification Scheme
 
-Items should be classified using existing item classification schemes, such as the [EC Common Procurement Vocabulary (CPV)](https://simap.ted.europa.eu/web/simap/cpv).
```

Comment:
@jpmckinney I also moved this sentence from here to the item.classification definition in the release-schema.